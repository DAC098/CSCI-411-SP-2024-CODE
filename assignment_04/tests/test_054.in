3
3 2 1
agtta gcagcccacacccacgc
caaaagagtaaa gagaaagataa
ttctgcgccatgttgg ggataacacaca
