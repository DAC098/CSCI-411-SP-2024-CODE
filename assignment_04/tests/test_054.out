__ag_______tta___
gcagcccacacccacgc
38
caaaagagtaaa
gagaa_agataa
6
ttctgcgccatgttgg
ggat__aaca__caca
17
